test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -19,6 +19,255 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
     {
+        public static TheoryData EmptyTagHelperBoundAttributeData
+        {
+            get
+            {
+                var factory = CreateDefaultSpanFactory();
+                var emptyAttributeError =
+                    "Attribute '{0}' on tag helper element '{1}' requires a value. Tag helper bound attributes of " +
+                    "type '{2}' cannot be empty.";
+                var boolTypeName = typeof(bool).FullName;
+
+                // documentContent, expectedOutput, expectedErrors
+                return new TheoryData<string, MarkupBlock, RazorError[]>
+                {
+                    {
+                        "<myth bound='' />",
+                        new MarkupBlock(
+                            new MarkupTagHelperBlock(
+                                "myth",
+                                new Dictionary<string, SyntaxTreeNode>
+                                {
+                                    { "bound", new MarkupBlock() }
+                                })),
+                        new[]
+                        {
+                            new RazorError(
+                                string.Format(emptyAttributeError, "bound", "myth", boolTypeName),
+                                absoluteIndex: 6, lineIndex: 0, columnIndex: 6, length: 5)
+                        }
+                    },
+                    {
+                        "<myth bound='    true' />",
```

Comment:
am I correct `<myth bound='    @true' />` leads to an error?  what about `<myth bound='    @(true)' />`?